src/dodal/beamlines/i22.py

```diff
@@ -22,13 +24,17 @@
 BL = get_beamline_name("i22")
 set_log_beamline(BL)
 set_utils_beamline(BL)
-"""VISIT is a placeholder value for reference.
-    Requires that all StandardDetector IOCs are running with /data mapping to /dls/i22/data
-    """
+
+# Currently we must hard-code the visit, determining the visit at runtime requires
+# infrastructure that is still WIP.
+# Communication with GDA is also WIP so for now we determine an arbitrary scan number
+# locally and write the commissioning directory. The scan number is not guaranteed to
+# be unique and the data is at risk - this configuration is for testing only.
 set_directory_provider(
     StaticVisitDirectoryProvider(
         BL,
-        "/data/i22/2024/VISIT",
+        Path("/dls/i22/data/2024/cm37271-2/bluesky"),
+        client=LocalDirectoryServiceClient(),
```

Comment:
Can LocalDirectoryServiceClient take a init arg of initial_count: int = 0 which can be set arbitrarily high to prevent collision? I worry about deploying something that can overwrite data to the beamline.

Reply:
Maybe I'm lacking context here but if blueapi is restarted mid-experiment I think this overwrites data - I think that's a pretty likely case for the June experiment...

As a simple interim solution before the proper GDA <-> bluesky communication is sorted, maybe we could mitigate this by the folder containing the timestamp at which `StaticVisitDirectoryProvider` was provided - then at least a subsequent restart wouldn't overwrite old data.

Reply:
This absolutely can overwrite data, see the comment in the diff. It meant to just be a temporary change and only write to `<commissioning dir>/bluesky`, so as not to break anything else. I felt that was sufficient protection for the beamline but if you two don't, no problem, we can think of something else. 

@Tom-Willemsen this is not intended for the June experiment, just to unblock testing in the interim so blueapi can at least write _some_ data. Creating a unique directory is an interesting idea, but blueapi doesn't have write permissions to the visit directory and in my opinion it never should, then we don't have to deal with the gda2 problem.